project.godot

```diff
@@ -52,6 +52,8 @@ enabled=PoolStringArray( "res://addons/colorful-palette-swap/plugin.cfg" )
 wind=false
 resol=false
 log=false
+debugger=false
+stdout=false
```

Comment:
What do these settings do? Will this turn off the gdscript debugger for everyone?